pkg/cgroups/systemd_linux.go

```diff
@@ -260,3 +262,35 @@ func resourcesToProps(res *configs.Resources, v2 bool) (map[string]uint64, map[s
 
 	return uMap, sMap, bMap, iMap, structMap
 }
+
+func RangeToBits(str string) ([]byte) {
```

Comment:
since it is used only here, can we make it private (`rangeToBits`)?

Reply:
I made it to private (`rangeToBits`).